ndbench-cli/build.gradle

```diff
@@ -30,7 +30,7 @@ dependencies {
     compile project(':ndbench-cass-plugins')
     // compile project(':ndbench-dynamodb-plugins') // Commented as part of https://github.com/Netflix/ndbench/issues/197
     compile project(':ndbench-cockroachdb-plugins')
-    compile project(':ndbench-dyno-plugins')
```

Comment:
I would put changes related to dyno plugin in the different commit but in same PR